packs/data/pfs-season-4-bestiary.db/determined-rhenei-1-2.json

```diff
@@ -0,0 +1,1992 @@
+                },
+                "school": {
+                    "value": "evocation"
+                },
+                "secondarycasters": {
+                    "value": ""
+                },
+                "secondarycheck": {
+                    "value": ""
+                },
+                "slug": "acid-splash",
+                "source": {
+                    "value": "Pathfinder Core Rulebook"
+                },
+                "spellType": {
+                    "value": "attack"
+                },
+                "sustained": {
+                    "value": false
+                },
+                "target": {
+                    "value": "1 creature"
+                },
+                "time": {
+                    "value": "2"
+                },
+                "traditions": {
+                    "custom": "",
+                    "value": [
+                        "arcane",
+                        "primal"
+                    ]
+                },
+                "traits": {
+                    "rarity": "common",
+                    "value": [
+                        "acid",
+                        "attack",
+                        "cantrip"
+                    ]
+                }
+            },
+            "type": "spell"
+        },
+        {
+            "_id": "eXsXQwNU2nuPb2hD",
+            "flags": {
+                "core": {
+                    "sourceId": "Compendium.pf2e.spells-srd.y6rAdMK6EFlV6U0t"
+                }
+            },
+            "img": "systems/pf2e/icons/spells/burning-hands.webp",
+            "name": "Burning Hands (Bludgeoning, Water)",
+            "sort": 400000,
+            "system": {
+                "ability": {
+                    "value": ""
+                },
+                "area": {
+                    "type": "cone",
+                    "value": 15
+                },
+                "category": {
+                    "value": "spell"
+                },
+                "components": {
+                    "focus": false,
+                    "material": false,
+                    "somatic": true,
+                    "verbal": true
+                },
+                "cost": {
+                    "value": ""
+                },
+                "damage": {
+                    "value": {
+                        "0": {
+                            "applyMod": false,
+                            "type": {
+                                "categories": [],
+                                "subtype": "",
+                                "value": "bludgeoning"
+                            },
+                            "value": "2d6"
+                        }
+                    }
+                },
+                "description": {
+                    "value": "<p>Gouts of flame rush from your hands. You deal 2d6 fire damage to creatures in the area.</p>\n<hr />\n<p><strong>Heightened (+1)</strong> The damage increases by 2d6.</p>"
```

Comment:
Should the description be updated?

Reply:
It is the sorcerer class feature that would modify it. The entry in the quest is written as Spell Name (bludgeoning, water) so the traits should be modified but the rest probably isn't worth changing